open-company/open-company-web
# Mobile google signin

_Test steps pending_

Review with:
- [Mobile](https://github.com/open-company/open-company-mobile/pull/2)
- [Auth](https://github.com/open-company/open-company-auth/pull/90)

~**NB:** this PR is based on #924, so we should review/merge that PR into `mainline`, and then `mainline` into _this_ PR before any review takes place.~ 

**iOS:** please use TestFlight build `1.1.2 (13)` to test this PR
**Android:** [Alpha opt-in link](https://play.google.com/apps/testing/io.carrot.mobile). `1.1.2 (13)` is the build you need here.

To test:
- Onboard user with Google on mobile
- [x] All clear?
- Onboard user with Google on desktop
- [x] All clear?
- Sign in with Google user on mobile
- [x] Good?
- Sign in with Google user on desktop
- [x] Good?

_The reason we test both mobile and desktop is that both apps use the `login-wall` component, so we'd like to test that desktop still retains the "old" login logic_
